tests/storage-vm

```diff
@@ -484,6 +484,20 @@ for poolDriver in $poolDriverList; do
 
         lxc delete -f v1
 
+        echo "==> Checking start / stop operations with root size.state being unset"
+        # We should be able to start a VM with migration.stateful=true eventhough the root `size.state` config is unset
+        lxc launch "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm --config migration.stateful=true -s "${poolName}"
```

Comment:
@simondeziel im going to merge this PR as I see this pattern occurring a lot.
But can we somehow avoid needing to specify the default value for TEST_IMG in every test and instead default to ubuntu-minimal-daily:22.04 in a single place where TEST_IMG is defined and just expect it to always be populated?

Reply:
Thanks!